code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -324,6 +324,9 @@
 			if(SHUTTLE_CALL)
 				if(!SSshuttle.canRecall())
 					dat += "<div class='notice red'>The station is currently undergoing evacuation procedures.</div><br>"
+	// SKYRAT EDIT ADDITION BEGIN
+	dat += "<div class='notice'>If you don't see any jobs, make sure you have at least 150 characters of Flavor Text written!</div>"
```

Comment:
Couldn't you just make it display that only if there's actually a failure with the flavor text requirement?